### PR TITLE
Implemented Vajra SA and TA Augments, added Vajra 75~99-4 to SQL

### PR DESCRIPTION
**Pull request by IxAalexI**
Also corrected some improper values.

### CODE REVIEW

**Comment by Hozu** [comment]
Just a heads up - your mod currently only affects melee attacks - not WSs. I made the same mistake when I implemented THF AF+1 hands.

**Comment by IxAalexI** [comment]
Thanks for the heads up, looking at it now to figure out where to stick the mods in.

File: src/map/attack.cpp [hunk 1]
```diff
@@ -363,9 +363,21 @@ void CAttack::ProcessDamage()
...
 	}
 
+    // Apply Sneak Attack Augment Mod
+    if (m_attacker->getMod(MOD_AUGMENTS_SA) > 0 && m_trickAttackDamage > 0 && m_attacker->StatusEffectContainer->HasStatusEffect(EFFECT_SNEAK_ATTACK))
+    {
+        m_damage += (m_damage * ((100+(m_attacker->getMod(MOD_AUGMENTS_SA)))/100));
+    }
+
+    // Apply Trick Attack Augment Mod
+    if (m_attacker->getMod(MOD_AUGMENTS_SA) > 0 && m_trickAttackDamage > 0 && m_attacker->StatusEffectContainer->HasStatusEffect(EFFECT_TRICK_ATTACK))
```
Comment (TeoTwawki) [inline src/map/attack.cpp]: I think you meant to use MOD_AUGMENTS_TA on that line.

Comment (IxAalexI) [inline src/map/attack.cpp]: Nah that's what I meant to do, both SA and TA return damage under m_trickAttackDamage. Looked odd to me too but that's what's already there.

Comment (TeoTwawki) [inline src/map/attack.cpp]: lol. I was like "but the comment says..." and then got to "nvm".

**Comment by IxAalexI** [comment]
Fixed

**Pull request merged**